### ISSUE TITLE
Cleanup temp files

### ISSUE THREAD

**Issue by durzo** [ecampbell/moodle-qformat_wordtable#5]
This plugin does not clean up the temporary files it makes, leaving $CFG->tempdir an absolute mess at the root level.

It would be good idea to use make_request_directory() rather than write directly to $CFG->tempdir, as it will get automatically removed by the shutdown handler.

**Comment by ecampbell** [comment]
Fair point, I've released v3.6.3 to stop leaving any temporary files lying around. I just realised that tempnam() actually creates the file, and I was renaming the returned filename for debugging purposes, which meant that the original created file was never deleted. I didn't use make_request_directory() yet, but it's a good suggestion and I'll incorporate it in the next major release, if I can figure out a way to keep the temporary files when debugging.
